pkgs/tools/misc/coreutils/default.nix

```diff
@@ -118,6 +118,12 @@ stdenv.mkDerivation rec {
       # TODO(19b98110126fde7cbb1127af7e3fe1568eacad3d): Needed for fstatfs() I
       # don't know why it is not properly detected cross building with glibc.
       "fu_cv_sys_stat_statfs2_bsize=yes"
+    ]
+    ++ optionals (stdenv.hostPlatform != stdenv.buildPlatform && stdenv.hostPlatform.isLinux) [
```

Comment:
We do now have a `isCross` variable in this scope, using that would be cleaner imo

Reply:
I don't think we should be checking whether we're cross-compiing at all.  This isn't a check it makes sense to do at build time in something like Nixpkgs where it's very likely the build machine is not representative of the machines the package will be run on anyway.

So I think it would be better to just set it for all Linux systems, whether cross-compiling or not.  Less cross specific code also means cross is less likely to get accidentally broken in future because it wasn't specifically tested.

Reply:
Alright, a staging PR it is